api/external/infra_proxy/request/clients.proto

```diff
@@ -18,3 +18,27 @@ message Client {
     // Client name.
     string name = 3;
 }
+
+message CreateClient {
+    // Chef organization ID.
+    string org_id = 1;
+    // Chef Infra Server ID.
+    string server_id = 2;
+	// Client name.
+	string name = 3;
+	// Boolean indicates client type is validator or not.
+	bool validator = 4;
+    // Boolean indicates whether it's required to create a key or not.
```

Comment:
make the proper alinement

Reply:
I guess alignment looks okay here, any specific ask?

Reply:
It looks OK in some editors and it looks wrong in some editors--that is because you are mixing spaces and tabs.  Please change to all spaces.
_(This applies to several files.)_

Reply:
Yeah, Due to some misconfiguration in my local VSCode. Have fixes for some files, will keep it for future in relevant files.
Thanks!